File: lab3c.c
        #include <stdio.h>
        #include <stdlib.h>

        struct aluno{

            int matricula;
            float prova[3], trabalho[2];

        };

        typedef struct aluno aluno;


        int main (int argc, char *argv[]){

            if (argc != 2)
            {
                printf("Numero de argumentos invalido!");
            }

            int N;
            int i;
            int j;
            FILE *fp;

            printf("Digite o numero de alunos:\n");
            scanf("%d", &N);

            aluno *estudante;

            estudante = (aluno *) malloc (N * sizeof(aluno));


            for(i = 0; i < N; i++){
                printf("Digite a matricula do aluno %d:\n", i+1);
                scanf("%d", &estudante[i].matricula);
                for(j = 0; j < 3; j++){
                    printf("Digite a nota da prova %d do aluno %d:\n", j + 1, i+1);
                    scanf("%d", &estudante[i].prova[j]);
                }
                for(j = 0; j < 2; j++){
                    printf("Digite a nota do trabalho %d do aluno %d:\n", j + 1, i+1);
                    scanf("%d", &estudante[i].trabalho[j]);
                }
                printf("\n");
            }

            fp =  fopen(argv[1], "w");

            fprintf(fp, "ALUNO \tMATRICULA \tTRABALHO 1 \tTRABALHO 2 \tPROVA 1 \tPROVA 2 \tPROVA 3\n\n");

             for (i = 0; i < N;i++)
            {
                fprintf(fp, "%d \t%d\t\t", i+1, estudante[i].matricula);
                
                for (j = 0; j < 3;j++)
                {
                    fprintf(fp, "%.2f \t\t", estudante[i].trabalho[j]);
                }
                
                for (j = 0; j < 2;j++)
                {
                    fprintf(fp, "%.2f \t\t", estudante[i].prova[j]);
                }
               
                fprintf(fp, "\n");
            }

            fclose(fp);

            free(estudante);

            system("PAUSE");

            return 0;

        }

        

File: lab3c.py
matricula = []
trab1 = []
trab2 =[]
p1 = []
p2 = []
p3 = []



n = input("Digite a quantidade de alunos:")

for i in range(int(n)):
    print("Digite a matricula do aluno ", i+1, ":")
    x = input()
    matricula.insert(int(n), int(x))

    print("Digite a nota do tabalho 1 do aluno ", i+1, ":")
    x = input()
    trab1.insert(int(n), x)

    print("Digite a nota do tabalho 2 do aluno ", i+1, ":")
    x = input()
    trab2.insert(int(n), x)

    print("Digite a nota da prova 1 do aluno ", i+1, ":")
    x = input()
    p1.insert(int(n), x)

    print("Digite a nota da prova 2 do aluno ", i+1, ":")
    x = input()
    p2.insert(int(n), x)

    print("Digite a nota da prova 3 do aluno ", i+1, ":")
    x = input()
    p3.insert(int(n), x)


fp = open("notas.txt", "w")

for i in range(int(n)):
    fp.write("%d " %matricula[i])
    fp.write("%.2f " %float(trab1[i]))
    fp.write("%.2f " %float(trab2[i]))
    fp.write("%.2f " %float(p1[i]))
    fp.write("%.2f " %float(p2[i]))
    fp.write("%.2f\n" %float(p3[i]))
